recipes/xbitinfo/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "xbitinfo" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xbitinfo-{{ version }}.tar.gz
+  sha256: 4f44f1a09221aceadb8f09bcd2ec9397be74da4158e6e39893df47b27b53ac69
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - setuptools >=30.3.0
+    - setuptools-scm
+    - setuptools_scm_git_archive
+    - pyjulia >=0.5.7
+    - julia
+  run:
+    - python >=3.8
+    - xarray
+    - julia
+    - pyjulia >=0.5.7
+    - tqdm
+    - numcodecs >=0.10.0
+    # extras::viz
+    - matplotlib-base
+    - cmcrameri
+    # extras::prefect
+    - prefect >=1.0.0
+
+test:
+  imports:
+    - xbitinfo
+  commands:
+    - pip check
+  requires:
+    - pip
```

Comment:
Probably want to add a test that you can import and run your package, since PyJulia+conda can be a bit finicky.